runelite-api/src/main/java/net/runelite/api/IconID.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018, Hydrox6 <ikada@protonmail.ch>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.api;
+
+/**
+ * Enum of all official icons that Jagex uses in chat.
+ */
+public enum IconID
+{
+	PLAYER_MODERATOR,
+	JAGEX_MODERATOR,
+	IRONMAN,
+	ULTIMATE_IRONMAN,
+	DMM_SKULL_5_KEYS,
+	DMM_SKULL_4_KEYS,
+	DMM_SKULL_3_KEYS,
+	DMM_SKULL_2_KEYS,
+	DMM_SKULL_1_KEYS,
+	SKULL,
+	HARDCORE_IRONMAN,
+	NO_ENTRY,
+	CHAIN_LINK;
+
+	@Override
+	public String toString()
+	{
+		return "<img=" + String.valueOf(this.ordinal()) + ">";

```

Comment:
Are you sure you want to use ordinals here? That seems like bad idea, mainly if we will be missing any icons in middle or want to add new icons at the end without adding any icons before that